### ISSUE TITLE
phpMyAdmin 4.9.8 throws syntax error on PHP 5.5

### ISSUE THREAD

**Issue by prbt2016** [phpmyadmin/phpmyadmin#17306]
### Describe the bug

I was in the process of installation of phpMyAdmin 4.9.8 on Centos 7 with Apache 2.2, PHP 5.5, MYSQL 5.5. 

However when I try to visit the URL to phpMyAdmin, following error is thrown i.e : 

Parse error: syntax error, unexpected '?' in /{{PATH}}/{{TO}}/{{PHPMYADMIN}}/libraries/classes/DatabaseInterface.php on line 1566 

On line 1566 the following code is causing the issue : 

` $storageDbName = $GLOBALS['cfg']['Server']['pmadb'] ?? '';`

The Null coalescing operator ?? was added since PHP 7.0 : 

https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op

Are the minimum requirements changed? 

Official site mentions that PHP supported is from 5.5 to 7.4 in the link below : 

https://www.phpmyadmin.net/files/4.9.8/

Could you please fix?

### To Reproduce

1. Install phpmyadmin on PHP 5.5
2. Click on URL to phpmyadmin i.e http://{{domain.com}}/phpmyadmin/
3. Above error is thrown

### Expected behavior

Login page should have been shown.

### Server configuration

- Operating system: Centos 7.6.1810 (Core)
- Web server: Apache 2.2.34
- Database version: 5.5.62
- PHP version: 5.5.38
- phpMyAdmin version: 4.9.8

### Client configuration

- Browser: 7.6.1810 (Core)
- Operating system: Centos 7.6.1810 (Core)

**Comment by ibennetch** [comment]
The requirements for 4.9.x haven't changed, it looks like we just made a mistake and should be releasing 4.9.9 soon to fix it.

@williamdes could you please take a look at this when you have a chance?

**Comment by mauriciofauth** [comment]
Hi @prbt2016.
Thank you for reporting this issue. This is a mistake that was introduced when we back-ported a fix into the 4.9.x branch. 
As an immediate fix, you can change these lines to:
```diff
diff --git a/libraries/classes/DatabaseInterface.php b/libraries/classes/DatabaseInterface.php
index 7ab5d754fb..0edc297cd3 100644
--- a/libraries/classes/DatabaseInterface.php
+++ b/libraries/classes/DatabaseInterface.php
@@ -1563,7 +1563,7 @@ class DatabaseInterface
      */
     public function initRelationParamsCache()
     {
-        $storageDbName = $GLOBALS['cfg']['Server']['pmadb'] ?? '';
+        $storageDbName = isset($GLOBALS['cfg']['Server']['pmadb']) ? $GLOBALS['cfg']['Server']['pmadb'] : '';
         // Use "phpmyadmin" as a default database name to check to keep the behavior consistent
         $storageDbName = $storageDbName !== null
                             && is_string($storageDbName)
@@ -1574,7 +1574,7 @@ class DatabaseInterface
         $this->relation->fixPmaTables($storageDbName, false);
 
         // This global will be changed if fixPmaTables did find one valid table
-        $storageDbName = $GLOBALS['cfg']['Server']['pmadb'] ?? '';
+        $storageDbName = isset($GLOBALS['cfg']['Server']['pmadb']) ? $GLOBALS['cfg']['Server']['pmadb'] : '';
 
         // Empty means that until now no pmadb was found eligible
         if (empty($storageDbName)) {
```

**Comment by prbt2016** [comment]
Hello @mauriciofauth ,

Thanks for the fix. Shall check this out.

Regards.
